es-design-system/pages/organisms/es-review.vue

```diff
@@ -8,6 +8,7 @@
                 :id="review.id"
                 :key="review.id"
                 user-id="3"
+                report-flag-visible
```

Comment:
Instead of hardcoding this, we want to set the value to true in the `data()` section of this file.